generator.go

```diff
@@ -188,7 +197,7 @@ func (g *Generator) GoCode(observedValue *ObservedValue) ([]byte, error) {
 }
 
 // GoType returns the Go type for o and whether it has been omitted.
-func (g *Generator) GoType(o *ObservedValue, observations int, imports map[string]bool) (string, bool) {
+func (g *Generator) GoType(o *ObservedValue, observations int, imports map[string]struct{}) (string, bool) {
```

Comment:
Although I agree that `map[string]struct{}` is a better representation to use for a set of imports, this API change breaks backwards compatibility, which would require a major version bump. So, please revert this to use `map[string]bool`.